### PR TITLE
Milestone 1: Complete Website

### PR DESCRIPTION
**Pull request by AsumVictor**
### Milestone 1: Complete AiAsum bootcamp website
I have successfully finished developing the capstone project, AiAsum boot camp, adhering to the specified [project requirements.](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/business_requirements.md)

I have added the following features: 
- [x] A responsive navigation bar on both the home and about pages.
- [x] Hero section for both about and home page.
- [x] A section on the home page to show the courses outline of the Bootcamp
- [x] A section on the home page to show the tutors of the Bootcamp
- [x] A section on the home and about page to show the Bootcamp's sponsors
- [x] There is a footer for both the home and about page
- [x] A section on the About page to display images and stories about past events of the Bootcamp
- [x] I separated the tutors' and courses' sections data to enhance organized work

I have ensured that:
- I created an Array to store information in the tutors and courses section
- I displayed the content in the tutors and courses section dynamically
- There are no linter errors in the project.
- Used Correct git-flow.
- Documented my work in a professional manner according to Professional code documenting requirement
- Included a descriptive Readme file.
- Used descriptive commit messages
- Followed the recommended HTML, CSS, and Js best practices

Best regards
Asum Victor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,168 @@
...
+- **About Page**
+- **Images and Story about past events**
+- **Tutors Section which contains tutors for the program**
+- **Get Coupon button to claim a discount**
+- **Courses Outline Section**
+
+## Live demo <a name="key-features"></a>
+
+I have deployed the project of github. [Click to view the online version of my project](https://asumvictor.github.io/AiAsum-bootcamp-website)
+
```
Comment (roshan-bajgain) [inline README.md:70]: - [x] Everything is good only small changes are required so I would suggest you please fix this Live demo so your README file looks professional.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      </p>
+      <div class="flex flex-col mt-16 w-full md:w-[730px]">
+        <p class=" text-xl md:text-3xl text-veryDarkBlue">
+          <time datetime="2023-06-20">June 20th, 2023 (MON)</time> to
+          <time datetime="2023-06-25">June 25th, 2023 (FRI)</time>
+        </p>
+        <p>
+        <address>@AiAsum Conference Center, Ho Akpenami Road</address>
+        </p>
+      </div>
```
Comment (roshan-bajgain) [inline index.html]: - [x] I would suggest you please use the exact or similar font if you notice then the navbar has a different font than yours. Please look at this issue and also the content or paragraph has different color or font. Please try to stick to the design as much as possible. Also, decrease the font weight.
- [x] If you notice that the content is aligned to the left in the Home page. Please fix this issue also.
- [x] I would suggest you please add more font-weight to h1 tag so it looks like the design.
- [x] In the design It's possible to navigate between the pages, there shoudl be link from `Home` page to `About` page and the opposite too. Please implement it as I notice there is only link for about page but when we click About page then there should be option for Home page also. It helps to make better UI
### Original design
original


### Your design
his

File: index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        <img src="./images/microsoft.png" alt="microverse" class="w-[9rem]">
+        <img src="./images/coursera-wordmark-blue.svg" alt="microverse" class="w-[9rem]">
+      </div>
+    </section>
+  </main>
+  <footer class="flex px-3 flex-col md:flex-row gap-8 py-[41px] items-center justify-center">
+    <a href="index.html" class="flex items-center">
+      <img src="/images/logo.png" alt="brand-logo" class="w-[50px]" />
+      <p class="text-2xl font-bold text-veryDarkBlue">
+        <span class="text-lightPink">I</span>Bootcamp
```
Comment (roshan-bajgain) [inline index.html:120]: - [x] Everything is awesome only small changes are required please decrease the font weight and try to use the same font so It matches the design.
- [x] Please make sure that title, subtitle, and description are aligned on the same line. Please see the image below for reference.
- [x] [Optional] I would suggest you please increase the height and width of the image to match the design.
### Original design
original

### Your design
his

File: about.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+      <a href="index.html" class="flex items-center">
+        <img src="/images/logo.png" alt="brand-logo" class="w-[50px]" />
+        <p class="text-2xl font-bold text-veryDarkBlue">
+          <span class="text-lightPink">I</span>Bootcamp
+        </p>
+      </a>
+      <address class="text-center">
+        2023 AiAsum Coding bootcamp Ghana. Some Rights Reserved
+      </address>
+    </footer>
```
Comment (roshan-bajgain) [inline about.html]: - [x] If you notice then you will find only one footer that is black on color please implement the footer in about.html to match the design. Also see image below for reference
### Your design
his

### Original design
original

File: about.html [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+          />
+          <img src="./images/microsoft.png" alt="microverse" class="w-[9rem]" />
+          <img
+            src="./images/coursera-wordmark-blue.svg"
+            alt="microverse"
+            class="w-[9rem]"
+          />
+        </div>
+      </section>
+    </main>
```
Comment (roshan-bajgain) [inline about.html]: - [x] Every content is awesome but you should stick to the design as much as possible. The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design. I would suggest you please use same font for the content to match the original design. You can use the google font and use lato with Thin 100 Italic. Please check this 
`<style>
@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,700;1,100&display=swap');
</style>`
### Original design
original

### Your design
his

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @AsumVictor, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (AsumVictor) [reply README.md:70]: Thank you.

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**